lib/complete.js

```diff
@@ -21,6 +25,7 @@ export default class Complete extends Domodule {
     }
     this.term = term;
     this.endpoint = options.endpoint || this.options.endpoint;
+    this.listClass = options.listClass || this.options.listClass;
```

Comment:
defaults were just added by @Belelros, we should impletment when domodule is released